plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -113,14 +119,14 @@ export function transformSeries(
   } else {
     plotType = seriesType === 'bar' ? 'bar' : 'line';
   }
-  const lineStyle = isConfidenceBand ? { opacity: 0 } : {};
-
+  const lineStyle = isConfidenceBand ? { opacity: 0 } : { opacity };
```

Comment:
I think we can use `OpacityEnum.Transparent ` to represent `0`.

Reply:
Thanks, good catch!